### ISSUE TITLE
Porting to TF: need some info on preprocessing

### ISSUE THREAD

**Issue by shkarupa-alex** [DingXiaoH/RepLKNet-pytorch#2]
Could you please tell what preprocessing did you used for train/eval models?

Normalization style:
- tf: img / 127.5 - 1.
- torch: (img / 255. - [0.485, 0.456, 0.406]) / [0.229, 0.224, 0.225]
- caffe: img - [103.939, 116.779, 123.68]

Image crop/resize:
- crop size if used
- resize interpolation: bilinear / bicubic

Did you use same preprocessing for 1k and 21k models?

**Comment by DingXiaoH** [comment]
Hi, I have updated the training code. Please check datasets.py
